driver-async/src/main/com/mongodb/async/client/MongoClientSettings.java

```diff
@@ -362,12 +379,25 @@ public ReadPreference getReadPreference() {
      * Gets the credential list.
      *
      * @return the credential list
+     * @deprecated Prefer {@link #getCredential()}
      */
+    @Deprecated
     public List<MongoCredential> getCredentialList() {
         return credentialList;
     }
 
     /**
+     * Gets the credential list.
+     *
+     * @return the credential list
+     * @since 3.6
+     */
+    public MongoCredential getCredential() {
```

Comment:
Is there a test for this?

Reply:
Yes, in MongoClientSettingsSpecification.groovy, line 170.  But there isn't one for MongoClientSettings.Builder#credential(MongoCredential), so adding one for that.